VirtoCommerce.Domain/Order/Model/CustomerOrder.cs

```diff
@@ -15,6 +15,7 @@ public class CustomerOrder : OrderOperation, IHaveTaxDetalization, ISupportSecur
         public string OrganizationName { get; set; }
         public string EmployeeId { get; set; }
         public string EmployeeName { get; set; }
+        public string OuterId { get; set; }
```

Comment:
Please add a comment to the new property.  "The order identifier in an external system."